__tests__/integrations.test.js

```diff
@@ -197,12 +200,77 @@ describe('Articles', () => {
     test('GET 400: Endpoint returns error message when article does not exist', () => {
       return request(app)
         .get('/api/articles/200/comments')
-        .expect(400)
+        .expect(404)
         .then(({ body }) => {
           const expectedResponse =
             'Sorry! That particular record was not found';
           expect(body.errorMessage).toBe(expectedResponse);
         });
     });
   });
+  describe('POST /api/articles/:article_id/comments', () => {
+    test('POST 200: Endpoint adds a new comment to the corresponding article', () => {
+      const newComment = {
+        username: 'butter_bridge',
+        body: 'example comment',
+      };
+      return request(app)
+        .post('/api/articles/2/comments')
+        .send(newComment)
+        .expect(201)
+        .then(({ body: { comment } }) => {
+          expectedResponse = {
+            comment_id: expect.any(Number),
+            body: newComment.body,
+            article_id: expect.any(Number),
+            author: newComment.username,
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+          };
+          expect(comment).toMatchObject(expectedResponse);
+        });
+    });
+    test('POST 400: Endpoint returns an error for an invalid article_id', () => {
```

Comment:
is this article id invalid or valid but non-existent?

Reply:
we should have a separate test for posting to invalid article id e.f. /api/articles/bananas/comments